tests/StringTemplate/SprintfEngineTest.php

```diff
@@ -44,12 +44,27 @@ public function testRenderWithObjectValues()
             )
         );
     }
+
+    public function testRenderWithAdditionalFormatting()
+    {
+        $engine = new SprintfEngine();
+        $this->assertEquals(
+            'I have 1.2 (1.230000E+0) apples.',
+            $engine->render(
+                "I have {num%.1f} ({num%.6E}) {fruit}.",
+                [
```

Comment:
We need to mantain php 5.3 compatibility even at the test level, can you please fix that? Thanks!